back/src/adapters/primary/scripts/seed.ts

```diff
@@ -21,6 +27,7 @@ const seed = async () => {
     await featureFlagsSeed(client);
     await agencySeed(uow, client);
     await establishmentAggregateSeed(uow, client);
+    await seedConventions(uow);
```

Comment:
on peut rester cohérent sur le nomage ? 
```suggestion
    await conventionsSeed(uow);
```

Ou alors renommer les autres ...